src/gromit-mpx.c

```diff
@@ -52,6 +53,7 @@ GromitPaintContext *paint_context_new (GromitData *data,
   context->arrowsize = arrowsize;
   context->minwidth = minwidth;
   context->paint_color = paint_color;
+  context->maxsize = maxsize;
```

Comment:
Please move this directly beneath `minwidth`.

Reply:
Done.